src/Module.php

```diff
@@ -24,11 +24,12 @@ public function getConfig()
     /**
      * Register a specification for the ValidatorManager with the ServiceListener.
      *
-     * @param \Zend\ModuleManager\ModuleEvent
+     * @param \Zend\ModuleManager\ModuleManager $moduleManager
      * @return void
      */
-    public function init($event)
+    public function init($moduleManager)
```

Comment:
Potentially. We've typically not done it, in order to prevent the inevitable "zend-modulemanager should be listed as a package requirement" issues. For some reason, not using a typehint doesn't trigger those remarks, even though the code ends up working exactly the same.